File: ws/chicaiza/u1/ws04_Dependencies/JavaApplication7/src/ec/espe/edu/chickenfarmsimulator/model/Egg.java
package ec.espe.edu.chickenfarmsimulator.model;

public class Egg {
    private char size;

    // Constructor vacío
    public Egg() {
    }

    // Constructor con parámetro
    public Egg(char size) {
        this.size = size;
    }

    // Getter
    public char getSize() {
        return size;
    }

    // Setter
    public void setSize(char size) {
        this.size = size;
    }

    // toString
    @Override
    public String toString() {
        return "Egg{" + "size=" + size + '}';
    }
}

File: hw/bonilla/u1/HW07ChickenFarmSimulatorWithInput/ChickenFarmSimulatorJavaScript/Egg.js
class Egg {
    constructor(size) {
        this.size = size;
    }

    toString() {
        return `Egg{size=${this.size}}`;
    }

    getSize() {
        return this.size;
    }

    setSize(size) {
        this.size = size;
    }
}